src/Framework/Support/Facades/CurrencyFacade.php

```diff
@@ -113,4 +124,20 @@ public function getBaseCurrency(): Currency
     {
         return new Currency(give_get_option('currency', 'USD'));
     }
+
+    /**
+     * Retrieves a list for all supported currencies.
+     *
+     * @unreleased
+     */
+    private function getCurrenciesList(): Currencies
+    {
+        return new AggregateCurrencies([
+            new ISOCurrencies(),
+            new BitcoinCurrencies(),
+            new CurrencyList([
+                'BTC' => 8,
```

Comment:
@JoshuaHungDinh The `BitcoinCurrencies()` doesn't have BTC??? I'm sure the answer is no if your'e adding this, I'm just shocked! 😅

Reply:
@JasonTheAdams Yea, BitcoinCurrencies() uses the official ISO code which is listed as XBT and not BTC.

Fun fact: The International Standards Organization (ISO) dictates that if a currency is not associated with a particular country, it should begin with an “X”. Bitcoin is standardized in the same way as gold, silver and palladium.

Reply:
Really!? Wow, that's pretty interesting. Learned something pretty interesting!

This does make me wonder, though. Does that mean that elsewhere, when `BTC` is being used as a currency, that's actually incorrect and should be `XBTC`?

Reply:
According to ISO, BTC is not a valid code. I suppose it's accepted worldwide as "BTC" because of its ticker symbol. You do bring up a good point. It is technically incorrect. A better option might be to update our references to the correct currency code rather than creating a custom list to accommodate an incorrect code. If we go that route, I do think we should handle updating the references in a separate PR and I can remove the custom list in this PR. This way we can still keep the aggregated list functionality but I defer to your call.

Reply:
You read my mind! I'd say let's add the `BitcoinCurrencies` list, remove the `BTC` one, and then correct where we're using BTC as a currency.